py4DSTEM/io/import_file.py

```diff
@@ -65,6 +65,8 @@ def import_file(
         data = read_gatan_K2_bin(filepath, mem=mem, binfactor=binfactor, **kwargs)
     # elif filetype == "kitware_counted":
     #    data = read_kitware_counted(filepath, mem, binfactor, metadata=metadata, **kwargs)
+    elif filetype == "mib":
+        data = load_mib(filepath, mem=mem, binfactor=binfactor, scan=(256,256),**kwargs)
```

Comment:
This seems to hardcode the scan shape to be 256x256, and as I understand it the MIB file does not always have the correct scan shape written inside of it (https://fpdpy.gitlab.io/fpd/convert_merlin_binary.html), so this should be a user-specifiable parameter

Reply:
Hi Steve(@sezelt ), 

Yes, Colin also raised this issue previously. I did this to use the general import function of py4DSTEM. 
`dataset = py4DSTEM.import_file(file_data)`
Since no other datasets take the scan value, I didn't know how to only selectively pass the scan value when a mib dataset is loaded. 
For now, the workaround is directly calling the internal loader. For example,
`dataset = py4DSTEM.io.load_mib(file_data,scan=(128,128))`.
This would work for a dataset with size (128,128). 

Please let me know if you have any other ideas I can circumvent this problem.

Reply:
the generic import function `py4DSTEM.import_file` accepts `**kwargs`, which are passed through to the specific readers. So you should just remove the explicit `scan=(256,256)` in this line, and instead within the `load_mib` function get the scan shape from its `**kwargs`. You can have default values as well, by doing something like `scan=kwargs.get("scan",(256,256))`

Reply:
Oh right! Yeah. I have made the changes to the code. Shall I create another pull request and close this one?

Reply:
Thanks! I have made the changes and pushed it again!